HW5/KaziutaRostyslav/hw5.py

```diff
@@ -0,0 +1,41 @@
+# list_num = [1,2,3,4,5,6,7,8,"9","10"]
+# float_list = []
+# for i in list_num:
+#     if isinstance(i,int):
+#         float_list.append(float(i))
+#
+# print(float_list)
+
+
+
+# x = int(input("Enter a number: "))
+# a, b = 0, 1
+# for i in range(1, x+1):
+#     print("{} => {}"format(i, a))
+#     a, b = b, a+b
+
+# a, b = 0, 1
+# i = 1
+# while a <= 13:
+#     print("{} => {}".format(i, a))
+#     a, b = b, a + b
+#     i += 1
+
+
+
```

Comment:
good job
look at this version of code
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)